Stratis.Bitcoin.Api/ApiFeature.cs

```diff
@@ -51,6 +53,14 @@ public override void Stop()
         {
             if (this.asyncLoop != null)
                 this.asyncLoop.Dispose();
+
+            // Make sure we are releasing the listening ip address / port
+            if (this.webHost != null)
+            {
+                this.logger.LogInformation($"Api stopping on url {this.fullNode.Settings.ApiUri}");
```

Comment:
pls no interpolated strings in logs + missing dot 
I'd also consider using proper use of uppercase chars when something goes to user (console) - i.e. API and URL instead of api and url

Reply:
This was for consistency with the pre-existing "starting" message. Both messages have now been changed..